osi_trafficsign.proto

```diff
@@ -9570,12 +9570,23 @@ message TrafficSign
         //
         VARIABILITY_OTHER = 1;
 
-        // Fixed sign, i.e. always present.
+        // Sign that can change neither semantically nor positionally, e.g.
+        // fixed immutable traffic sign.
         //
         VARIABILITY_FIXED = 2;
 
-        // Temporary or variable sign, e.g. on a sign bridge.
+        // Sign that can change semantically and positionally.
```

Comment:
As suggested in harmonization review, switch this to be undetermined legacy value, and point to new more specififc values:

```suggestion
        // Sign that can change semantically and/or positionally, i.e.
        // it is left unspecified which of those apply. Note that this
        // represents a legacy definition, please use the more specific
        // values below, if possible, to indicate the exact nature of
        // variability.
```